arrow-libs/plugins/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
```

Comment:
I guess this showed up due to the empty `settings.gradle.kts` I added? I added that because otherwise, I couldn't open the `plugins` folder in IntelliJ to get tooling to work.

Reply:
If these duplicated wrappers files can be removed that'd be great! There is a couple more in nested folders in `arrow-libs` I think :/

Reply:
Ok, we can look at this on a separate ticket then. It's interesting that IntelliJ is required to add it. 

I've created a [ticket](https://github.com/arrow-kt/arrow/issues/2531) to investigate this separately :slightly\_smiling\_face: